README.md

```diff
@@ -127,127 +127,205 @@ future state of the libraries and to discover projects for contribution.
+through and it's the driver's responsibility to update the checklist and assign
+the correct label to on the RFC throughout the process.
+
+[tracking issue template]: https://github.com/aws/aws-cdk-rfcs/blob/master/.github/ISSUE_TEMPLATE/tracking-issue.md
+
+When the issue is created, it is required to fill in the following information:
+
+1. **Title**: the name of the feature or change - think changelog entry.
+2. **Description**: a _short_ description of feature, as if it was already implemented.
+3. **Proposed by**: fill in the GitHub alias of the person who proposed the idea
+   under "Proposed by".
+
+### 2. API Bar Raiser
+
+The public API of a feature represents the surface through which users interact
+with it, and we want to make sure these APIs are consistent, ergonomic and
+designed based on the intent and the mental model of our users. Additionally,
+once we announce that a feature is "stable" (1.0, GA, etc) any breaking change
+to its public API will require releasing a new major version, so we think of API
+decisions as "one way doors".
+
+For each RFC, the core team will assign an **API Bar Raiser** who reviews and
+approves any public API of the feature. API Bar Raisers have veto rights on
+API-related design decisions, such as naming, structure, options, CLI commands
+and others.
+
+After an RFC is proposed, the core team will assign it an API Bar Raiser using a
+tiering model generally based on the size of the user base that will likely get
+exposed to the feature. As a general rule, the more "significant" the change is,
+we will assign a bar raiser with a wider and longer-term context of the project.
+
+API Bar Raisers are required to **sign off** on the **Working Backwards**
+section of your RFC, so we encourage to engage with them early in the process to
+make sure you are aligned on how the API should be designed.
+
+The API Bar Raiser is called out in the RFC tracking issue and in the RFC
+document itself. Feel free to reach us at the
+[#aws-cdk-rfcs](https://cdk-dev.slack.com/archives/C025ZFGMUCD) channel in the
+[cdk.dev](https://cdk.dev) slack workspace if you need a bar raiser assigned to
+your RFC.
+
+> The engineering solution proposed in an RFC does not need require approval
+> beyond the normal pull request approval model (e.g. a core team member needs
+> to approve the RFC PR and any subsequent changes to it).
+
+### 3. Organize a kick-off meeting
+
+Before diving into writing the RFC, it is highly recommended to organize a
+kick-off meeting that includes the API Bar Raiser and any stakeholders that
+might be interested in this RFC or can contribute ideas and direction. The goal
+of the meeting is to discuss the feature, its scope and general direction for
+implementation.
+
+Our experience shows that such a meeting can save a lot of time and energy.
+
+Use the RFC tracking issue to record some initial API and design ideas and
+collect early feedback and use cases as a preparation for the kick off meeting.
```

Comment:
I generally prefer doing this in the RFC rather than tracking issue. Basically a rough half baked (or not baked) rfc. 

It allows me to iterate on my idea via git, before I'm ready to share.

Can we say either or move to that model? I generally find doing this on issues to be a little hard to manage.

Reply:
I think some initial iteration on the issue is desirable (and actually came up in our retrospective as a light-weight process to get started).

Reply:
Changed to "you can..."